docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.7'
```

Comment:
Is the `docker-compose.yml` used for something when the project is deployed?

Reply:
I don't understand this question

Reply:
The question is really: Can the compose file be removed without removing any functionality? Because if it can, please remove it.

Reply:
It will be responsible for creating the image that then will be fetched when we want to spin up an instance of this program

Reply:
That's super legit. Thank you!